app/templates/index.js

```diff
@@ -1,3 +1,6 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
```

Comment:
Wow I am smoking crack. We did not add it to feathers-permissions. Sorry too much in flight yesterday. Will remove and ship.